examples/notebooks/voltage-regulator/publish_waveforms.ipynb

```diff
@@ -12,48 +12,118 @@
   },
   {
    "cell_type": "markdown",
-   "id": "db8657b1",
+   "id": "a0ec35e9",
    "metadata": {},
    "source": [
-    "## Set up the test result for publishing data\n",
+    "# Set up the test station and other metadata (one time setup)\n",
     "\n",
-    "First, we're going to set up our test result metadata and get everything ready to publish"
+    "Setting up the metadata for operators, test stations, hardware, etc will typically be done once. We establish 'aliases' which are human-readable strings that can be used to refer to metadata later."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "5aaa09bb",
+   "id": "72711880",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from ni.datastore import Client, Operator, TestResult, TestStation\n",
+    "from ni.datastore import (\n",
+    "    Client,\n",
+    "    HardwareItem,\n",
+    "    Operator,\n",
+    "    SoftwareItem,\n",
+    "    TestStation,\n",
+    "    Uut,\n",
+    "    UutInstance,\n",
+    ")\n",
+    "\n",
+    "# Constants\n",
+    "ALIAS_JAMES_BOWERY = \"Operator_BoweryJ\"\n",
+    "ALIAS_TEST_STATION_12 = \"TestStation_12\"\n",
+    "ALIAS_UUT_A861_42367 = \"UUT_A861_42367\"\n",
+    "ALIAS_SOFTWARE_ITEM_WINDOWS_10 = \"Windows_10.0.19044\"\n",
```

Comment:
Windows 10 support ends October 14. Consider upgrading your test station to Windows 11.